aviator-co/av
# [MER-927] auto open browser config

Added top level config value for controlling if the auto-open browser behavior from MER-874

# Test Plan
- add `openbrowser` false to `~/.av/config.yaml`
- open new PR on test repo and see that browser now does not open

# Results
Added `openbrowser: false` to config file and ran the following which -> browser did not open:

<img width="942" alt="Screen Shot 2022-06-23 at 2 11 57 PM" src="https://user-images.githubusercontent.com/107422947/175366871-9daad509-c191-4979-b2be-68aba32fa7b0.png">
